pyproject.toml

```diff
@@ -35,7 +35,8 @@ classifiers = [
     "Operating System :: MacOS"
 ]
 dependencies = [
-    "numpy >= 1.23.5"
+    "numpy >= 1.23.5",
+    "jax"
```

Comment:
I don't think we want this as an install dependency, but instead something that, if available, can be used. We certainly don't want to force users to install JAX if they have no intention of using roughpy JAX